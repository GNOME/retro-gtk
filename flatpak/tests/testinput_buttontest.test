[Retro Reftest]
Path=/testinput_buttontest
Core=/app/lib/libretro/testinput_buttontest_libretro.so

[Frame 0]
Tests=Run;Video;
Video=testinput_buttontest.png
