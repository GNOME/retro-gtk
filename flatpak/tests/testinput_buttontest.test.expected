/testinput_buttontest/Boot: OK
/testinput_buttontest/0/Run: OK
/testinput_buttontest/0/Video: OK
/testinput_buttontest/0/NoStateAccess: OK
